testplan/web_ui/testing/src/AssertionPane/AssertionTypes/GraphAssertions/DiscreteChartAssertion.js

```diff
@@ -0,0 +1,62 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import 'react-vis/dist/style.css';
+import './App.css';
+import * as GraphUtil from './graphUtils';
+import {
+  RadialChart
+} from 'react-vis';
+
+/**
+ * Component that are used to render a Chart (Data visualisations that don't require an XY axis).
+ */
+
+class DiscreteChartAssertion extends Component  {
+  components = {
+    Pie: RadialChart
+    }
+
+
+  render(){
+    let data = this.props.assertion.graph_data;
+    const graph_type = this.props.assertion.graph_type
+    const GraphComponent = this.components[graph_type];
+    let series_options = this.props.assertion.series_options;
+   // let graph_options = this.props.assertion.graph_options;
```

Comment:
What is happening with graph_options?

Reply:
There are currently no global graph options implemented for discrete charts (no axis labeling or legends etc), so it is an unused variable at the moment